spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTemplateIntegrationTests.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.spanner.core;
```

Comment:
Should these be created in an `it` test folder?

Reply:
I have no strong opinion on that, we can.
I put them next to the main beans where they are, template tests next to the SpannerTemplate, repository tests next to the SpannerRepository. Do you suggest it should be in 
`org.springframework.cloud.gcp.data.spanner.core.it` or `org.springframework.cloud.gcp.data.spanner.it`?

Reply:
I'm not sure to what extent best practices are involved here, but I've seen it as a rule in other projects to have a separate folder for integration tests.
Actually, for Pub/Sub we already started calling integration tests folders `integration`, so
`org.springframework.cloud.gcp.data.spanner.core.integration` could do it.